docs/source/tutorial/package_a_project.md

```diff
@@ -111,29 +111,19 @@ The resulting `.whl` packages only contain the Python source code of the Kedro p
 
 The project configuration is provided separately in a `tar.gz` file, also inside the `dist` folder. This compressed version of the config files excludes any files inside the `local` directory.
 
-### Package recipients
+### Run a packaged project
 
-Recipients of the `.whl` file need to have Python and `pip` on their machines, but do not need to have Kedro installed.
-
-A recipient can install the project by calling:
+To run a packaged project it must first be installed. To install the project, run the following command:
 
 ```bash
 pip install <path-to-wheel-file>
 ```
 
-An executable, `spaceflights`, is placed in the `bin` subfolder of the Python install folder, so the project can be run as follows:
-
-```bash
-python -m spaceflights
```

Comment:
I am a bit hesitant about removing the mention of `python -m <package>` here.

1. We still have `python -m package` mentioned in a few place
2. This seems to be going to be the way how we deploy kedro, at the moment we have issues with a `click` so it doesn't work well on Databricks, but it's a special case. I personally

Reply:
Both this way and the way I describe are equivalent. They both call `__main__:main`. In fact, both expose the `click` options for `kedro run`. Try running `python -m <package_name> --help` and checking it for yourself.

I'm not too concerned about Databricks, as this PR also adds clear instructions for running a packaged project on that platform.

That being said, I've added this back in.